easybuild/easyconfigs/g/GLFW/GLFW-3.2.1-intel-2018a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'GLFW'
+version = '3.2.1'
+
+homepage = 'https://www.glfw.org'
+description = '''GLFW is an Open Source, multi-platform library for OpenGL,
+    OpenGL ES and Vulkan development on the desktop'''
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['e10f0de1384d75e6fc210c53e91843f6110d6c4f3afbfb588130713c2f9d8fe8']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('X11', '20180131'),
```

Comment:
Is there really no dependency on Mesa/libGLU here?

Reply:
I think not (but am not an expert).  You can see more here: https://www.glfw.org/docs/3.3/compile.html

If you think it safer to add Mesa as an explicit dependency anyway, then I'm happy to do so.

Reply:
I'm not an expert either, but checking the link above and the sources it seems Mesa can be used at runtime if you build with `-DGLFW_USE_OSMESA`, as Mesa is built with gallium-osmesa.
however, in that case it is built without X11 support, so I think it is fine as it is without Mesa.